src/DataTable/TableActions.jsx

```diff
@@ -8,10 +8,21 @@ import Actions from './CollapsibleButtonGroup';
 const TableActions = ({ className }) => {
   const tableInstance = useContext(DataTableContext);
   const { tableActions } = tableInstance;
+  if (typeof tableActions === 'function') {
+    return <div className={classNames('pgn__table-actions', className)}>{tableActions()}</div>;
```

Comment:
Should `tableActions()` pass the `tableInstance` as an argument since the individual actions are passed `tableInstance` as well?